src/cache/redisCache.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { CACHE_MANAGER } from '@nestjs/cache-manager';
+
+@Injectable()
+export class RedisCacheService {
+  constructor(@Inject(CACHE_MANAGER) private readonly cache: Cache) {}
+
+  async get(key: string): Promise<any> {
+    return await this.cache.get(key);
```

Comment:
Redundant `await` if `this.cache.get` returns a Promise already
```suggestion
    return this.cache.get(key);
```